tools/api-markdown-documenter/api-report/api-markdown-documenter.alpha.api.md

```diff
@@ -344,6 +347,9 @@ function filterItems(apiItems: readonly ApiItem[], config: Required<ApiItemTrans
 // @public
 export function getApiItemTransformationConfigurationWithDefaults(inputOptions: ApiItemTransformationConfiguration): Required<ApiItemTransformationConfiguration>;
 
+// @public
+function getCustomBlockComments(apiItem: ApiItem): ReadonlyMap<string, DocSection[]>;
```

Comment:
if this is intended to be read-only, the arrays in the map should also be readonly.